src/main/java/hudson/plugins/jacoco/JacocoPublisher.java

```diff
@@ -34,348 +44,445 @@
+    }
 
-	public String getMaximumInstructionCoverage() {
-		return maximumInstructionCoverage;
-	}
+    @DataBoundSetter
+    public void setMinimumBranchCoverage(String minimumBranchCoverage) {
+        this.minimumBranchCoverage = checkThresholdInput(minimumBranchCoverage);
+    }
 
+    @DataBoundSetter
+    public void setMinimumComplexityCoverage(String minimumComplexityCoverage) {
+        this.minimumComplexityCoverage = checkThresholdInput(minimumComplexityCoverage);
+    }
 
+    @DataBoundSetter
+    public void setMinimumLineCoverage(String minimumLineCoverage) {
+        this.minimumLineCoverage = checkThresholdInput(minimumLineCoverage);
+    }
 
-	public String getMaximumBranchCoverage() {
-		return maximumBranchCoverage;
-	}
+    @DataBoundSetter
+    public void setMinimumMethodCoverage(String minimumMethodCoverage) {
+        this.minimumMethodCoverage = checkThresholdInput(minimumMethodCoverage);
+    }
 
+    @DataBoundSetter
+    public void setMinimumClassCoverage(String minimumClassCoverage) {
+        this.minimumClassCoverage = checkThresholdInput(minimumClassCoverage);
+    }
 
+    @DataBoundSetter
+    public void setMaximumInstructionCoverage(String maximumInstructionCoverage) {
+        this.maximumInstructionCoverage = checkThresholdInput(maximumInstructionCoverage);
+    }
 
-	public String getMaximumComplexityCoverage() {
-		return maximumComplexityCoverage;
-	}
+    @DataBoundSetter
+    public void setMaximumBranchCoverage(String maximumBranchCoverage) {
+        this.maximumBranchCoverage = checkThresholdInput(maximumBranchCoverage);
+    }
 
+    @DataBoundSetter
+    public void setMaximumComplexityCoverage(String maximumComplexityCoverage) {
+        this.maximumComplexityCoverage = checkThresholdInput(maximumComplexityCoverage);
+    }
 
+    @DataBoundSetter
+    public void setMaximumLineCoverage(String maximumLineCoverage) {
+        this.maximumLineCoverage = checkThresholdInput(maximumLineCoverage);
+    }
 
-	public String getMaximumLineCoverage() {
-		return maximumLineCoverage;
-	}
+    @DataBoundSetter
+    public void setMaximumMethodCoverage(String maximumMethodCoverage) {
+        this.maximumMethodCoverage = checkThresholdInput(maximumMethodCoverage);
+    }
 
+    @DataBoundSetter
+    public void setMaximumClassCoverage(String maximumClassCoverage) {
+        this.maximumClassCoverage = checkThresholdInput(maximumClassCoverage);
+    }
 
+    @DataBoundSetter
+    public void setChangeBuildStatus(boolean changeBuildStatus) {
+        this.changeBuildStatus = changeBuildStatus;
+    }
 
-	public String getMaximumMethodCoverage() {
-		return maximumMethodCoverage;
-	}
+    @DataBoundSetter
+    public void setInclusionPattern(String inclusionPattern) {
+        this.inclusionPattern = inclusionPattern;
+    }
 
+    @DataBoundSetter
+    public void setExclusionPattern(String exclusionPattern) {
+        this.exclusionPattern = exclusionPattern;
+    }
 
+    /** End Setters for Workflow **/
```

Comment:
Well, it's not for workflow. It's in general a good practice to use `@DataBoundSetter`s for fields that have a default value.